lib/acts_as_rdfable/migration_annotations.rb

```diff
@@ -61,4 +71,8 @@ def delete_column_annotation(table, column)
   def irreversible!(msg)
     raise ActiveRecord::IrreversibleMigration, msg if reverting?
   end
+
+  def dump_rdf_annotations
+    ::SeedDump.dump(RdfAnnotation, file: 'db/seeds/rdf_annotations.rb')
```

Comment:
Would it be possible to make the gem read from an initializer to set the value for the seed dump path?
If it is too much work it wouldn't be a deal-breaker, just thinking of avoiding hard-coded values.

Reply:
Probably?  Do you know of a good example to base this on?

Reply:
This looks like an okay pattern to copy: https://github.com/kaminari/kaminari/blob/00657f252244413b512349f4de20c63ee478baa5/kaminari-core/lib/kaminari/config.rb

Reply:
Added this.  Let me know what you think!